CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## Unreleased
+ - Added alert role to error message divs beneath card inputs
```

Comment:
A couple nitpicky suggestions for the Changelog:
- Looks like we use present tense verbs throughout the changelog. So in this case `Added` -> `Add`
- We usually add a reference to the issue we are addressing

```suggestion
 - Add alert role to error message divs beneath card inputs. (#845 [https://github.com/braintree/braintree-web-drop-in/issues/845])
```